aspnet/publishing/linuxproduction.rst

```diff
@@ -174,3 +151,83 @@ You can redirect application logs (``STDOUT`` and ``STERR``) in the program sect
 
     tail -f /var/log/hellomvc.out.log
 
+Securing our application
+------------------------
+
+Enable ``AppArmor``
+~~~~~~~~~~~~~~~~~~~
+Linux Security Modules (LSM) is a framework that is part of the Linux kernel since Linux 2.6 that supports different implementations of security modules. ``AppArmor`` is a LSM that implements a Mandatory Access Control system which allows you to confine program to a limited set of resources. Ensure `AppArmor <https://wiki.ubuntu.com/AppArmor>`__ is enabled and properly configured.
+
+Configuring our firewall
+~~~~~~~~~~~~~~~~~~~~~~~~
+Close off all external ports that are not in use. Uncomplicated firewall (ufw) provides a frontend for ``iptables`` by providing a command-line interface for configuring the firewall. Ensure that ``ufw`` is installed and available and configured allow traffic on any ports you may need.
```

Comment:
>installed and available and configured
can't we just say configured and enabled?

maybe:
Verify that ``ufw`` is configured to allow traffic on ports you need.